grapher/mapCharts/MapTooltip.tsx

```diff
@@ -2,113 +2,118 @@ import * as React from "react"
 import { computed } from "mobx"
 import { observer } from "mobx-react"
 import { Tooltip } from "../tooltip/Tooltip"
-import { takeWhile, last, first, isMobile } from "../../clientUtils/Util"
-import {
-    SparkBars,
-    SparkBarsDatum,
-    SparkBarsProps,
-} from "../sparkBars/SparkBars"
-import { SparkBarTimeSeriesValue } from "../sparkBars/SparkBarTimeSeriesValue"
-import { MapChartManager, ChoroplethSeries } from "./MapChartConstants"
-import { ColorScale } from "../color/ColorScale"
+import { MapChartManager } from "./MapChartConstants"
+import { ColorScale, ColorScaleManager } from "../color/ColorScale"
 import { Time } from "../../coreTable/CoreTableConstants"
 import { ChartTypeName } from "../core/GrapherConstants"
 import { OwidTable } from "../../coreTable/OwidTable"
-import { CoreColumn } from "../../coreTable/CoreTableColumns"
+import { EntityName, OwidVariableRow } from "../../coreTable/OwidTableConstants"
+import { LineChart } from "../lineCharts/LineChart"
+import { PrimitiveType } from "../../clientUtils/owidTypes"
+import classNames from "classnames"
+import { Bounds } from "../../clientUtils/Bounds"
+import { LineChartManager } from "../lineCharts/LineChartConstants"
+import {
+    darkenColorForHighContrastText,
+    darkenColorForLine,
+    darkenColorForText,
+} from "../color/ColorUtils"
+import { getPreposition } from "../../coreTable/CoreTableUtils"
+import { isNumber, NoUndefinedKeys } from "../../clientUtils/Util"
 
 interface MapTooltipProps {
-    tooltipDatum?: ChoroplethSeries
-    tooltipTarget?: { x: number; y: number; featureId: string }
-    isEntityClickable?: boolean
+    entityName: EntityName
     manager: MapChartManager
-    colorScale?: ColorScale
+    colorScaleManager: ColorScaleManager
+    formatValue: (d: PrimitiveType) => string
+    timeSeriesTable: OwidTable
+    tooltipTarget: { x: number; y: number; featureId: string }
     targetTime?: Time
+    isEntityClickable?: boolean
 }
 
+const SPARKLINE_WIDTH = 140
+const SPARKLINE_HEIGHT = 60
```

Comment:
One thing I would love to see is for these to be a little bit larger on a big wide screen.
I realize that the current size is a great compromise that works on both mobile and desktop, but on a large screen it can be hard to make out details.
Additionally, the small size leads to information like the x ticks and the min/max y ticks not being shown for lack of space.

Reply:
Testing this at 50% zoom level (don't have a big screen right now), the tooltip is still pretty massive?

<img width="604" alt="Screenshot 2022-02-16 at 19 35 09" src="https://user-images.githubusercontent.com/1308115/154343933-cc20da48-2d08-4648-a153-160796ebebbb.png">

Compared to the other elements (subtitle font size, legend), it seems a reasonable size?

If we made it much bigger, it would occlude way too big of an area of the map? I already think it's too big. 

Here's how it is currently as a comparison:

<img width="604" alt="Screenshot 2022-02-16 at 19 35 21" src="https://user-images.githubusercontent.com/1308115/154344159-a997bcf0-4f04-4a3b-9bfe-cc15e8a3f465.png">

Reply:
I'll deploy this (since it's an improvement over what we have today) but happy to continue discussing/tweaking the design later on.